handlers.go

```diff
@@ -74,6 +75,22 @@ func logSigningRequestFailure(sigreq formats.SignatureRequest, sigresp formats.S
 	}).Info(fmt.Sprintf("signing operation failed with error: %v", err))
 }
 
+// If the x5u is a local file, mirror it via the `x5u` endpoint instead.
+func mirrorLocalX5U(r *http.Request, response formats.SignatureResponse) string {
```

Comment:
I'm getting this confused with other similarly named things. Perhaps `rewriteX5UIfLocalFile`? Something about rewriting, maybe. The `mirror` sounds like it's copying the object from one place to another.

Also, nit: [godoc format](https://go.dev/blog/godoc) is to start the first sentence with the function's name